GitCommands/PathUtil.cs

```diff
@@ -332,5 +332,28 @@ string FindFile(string location, string fileName1)
                 return null;
             }
         }
+
+        [ContractAnnotation("=>false,errorMessage:null")]
+        [ContractAnnotation("=>true,errorMessage:notnull")]
+        public static bool TryDeleteDirectory([NotNull] this string path, out string errorMessage)
+        {
+            errorMessage = string.Empty;
```

Comment:
Nit: Currently, this function always sets a non-null `errorMessage`.

Reply:
Aye!